### PR TITLE
Add PowerPC:ToTest

### PR DESCRIPTION
**Pull request by k0da**
Signed-off-by: Dinar Valeev <dvaleev@suse.com>

### CODE REVIEW

File: totest-manager.py [hunk 1]
```diff
@@ -240,26 +243,30 @@ def factory_snapshottable(self):
 
         if not self.all_repos_done('openSUSE:%s' % self.project):
             return False
+        if self.arch == "x86_64":
+            for product in ['_product:openSUSE-ftp-ftp-i586_x86_64',
```
Comment (coolo) [inline totest-manager.py]: shouldn't this list of products come out of the ToTest class instead of depending on the arch?

**Comment by k0da** [comment]
ok updated.

@DimStar77 what do you think about getting rid of arch specific livecd products?

**Comment by DimStar77** [comment]
@k0da The LiveCDs are still one of the quickest way to get anything tested and demoed (their quality is not the best ever, but the effect of showing something is still there).

We could probably away with only one arch instead of multiple, but I don't think we should sacrifice it completely at this moment.

**Comment by coolo** [comment]
he talks about the .i586 in the package name

**Comment by DimStar77** [comment]
Ah; that actually makes sense: looking at openSUSE:Factory:Live/kiwi-image-livecd-kde.x86_64 for example, it's a unmodified link to openSUSE:Factory:Live/kiwi-image-livecd-kde.i586 (minus the other .spec files)... so splitting it in two does not really make a lot of sense

**Comment by k0da** [comment]
@DimStar77 so we're fine then? could you please review my changes?

**Comment by DimStar77** [comment]
Seems all right to me

**Comment by coolo** [comment]
will you also fix the Live project?

**Comment by k0da** [comment]
@DimStar77 ^^

**Comment by DimStar77** [comment]
I can do the changing of the names (dropping the .x86_64 packages linking) and renaming the .i586 to not have a suffix; but please only after the 0318 moves over to QA

**Comment by coolo** [comment]
that's what I thought

**Pull request merged**